docs/INSTALL.md

```diff
@@ -110,7 +110,6 @@ To configure/install the UPF in DPDK mode, the following changes are required:
      # In the order of (s1u sgi)
     -macaddrs=(9e:b2:d3:34:ab:27 c2:9c:55:d4:8a:f6)
     +macaddrs=(b4:96:91:b1:ff:f0 b4:96:91:b1:ff:f1)
-
```

Comment:
Why does this empty line need to be removed? It is part of the output of git diff

Reply:
Not sure how it got removed. will keep empty line